data/assets/scene/digitaluniverse/supernovaremnants.asset

```diff
@@ -33,19 +33,15 @@ local Object = {
     PolygonSides = 7,
     Unit = "pc",
     SizeSettings = {
-      ScaleExponent = 18.4,
+      ScaleExponent = 18.1,
```

Comment:
To discuss: should this size be reset as well?  and part of the general brightness at milkyway VS nightsky level discussion